core-web/apps/dotcms-ui/src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts

```diff
@@ -134,6 +134,23 @@ export class DotCrumbtrailService {
                     });
                 }
 
+                // To replace Browser URL with Pages URL
+                if (sections[0] == 'edit-page') {
+                    // Delete Site link
+                    crumbTrail = crumbTrail.filter((crumb) => crumb.label != 'Site');
```

Comment:
I think we can't compare with the string directly because this could be any name, can we do it with the url?

Reply:
Yes, we can use the URL. I think is safer that way